ci/release-info-v3.py

```diff
@@ -0,0 +1,100 @@
+import json
+import os
+import re
+import sys
+from datetime import date
+from subprocess import Popen, PIPE
+
+def check_version(ver):
+    # Checks package version for matching with the current tag reference
+    # print(ver[0])
+    if ref is not None and ref != 'refs/tags/v' + str(ver[0]):
+        return 0
+    else:
+        return 1
```

Comment:
This duplicated code could be avoided if making an `__init__.py` file in the `ci` folder and transforming it into a module iirc.

Reply:
I'd prefer to have a one script to rule the whole CI pipeline for making the release and then packages, not to split'em or use __init__, this is uneeded complication, so I'd prefer to leave it as it was in very first version of script.